bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Download.java

```diff
@@ -0,0 +1,102 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2018 Adobe Systems Incorporated
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+package com.adobe.cq.wcm.core.components.models;
+
+import com.adobe.cq.export.json.ComponentExporter;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Sling model for the Download component.
+ */
+public interface Download extends ComponentExporter {
+
+    String PN_TITLE_FROM_ASSET = "titleFromAsset";
```

Comment:
Same as above, I'd call it `useTitleFromAsset`.

Reply:
@jckautzmann - I think it's clear enough, and it follows our naming in the teaser [0]. What we are missing here are some docs for these properties as we have in the other component interfaces.

[0] https://github.com/adobe/aem-core-wcm-components/blob/master/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Teaser.java#L109